docs/reference/context.md

```diff
@@ -59,11 +59,11 @@ context is known. The body of a context expression may be evaluated zero, one,
 or multiple times, depending on how many different places it is put into.
 
 ## Location context
-Context can not only give us access to set rule values. It can also let us know
+Context cannot only give us access to set rule values. It can also let us know
```

Comment:
This one is actually wrong, see https://english.stackexchange.com/questions/531272/cannot-in-not-only-but-also

Reply:
Oops. Yes. Sorry.

```suggestion
Context can, not only, give us access to set rule values. It can also let us know
```

Reply:
notes for myself for later https://github.com/typst/typst/compare/main...jsoref:typst:fix-can-not-only

Reply:
I'll incorporate a fix into https://github.com/typst/typst/pull/5089